src/AccelerationPlanner.cpp

```diff
@@ -181,14 +181,29 @@ bool AccelerationPlanner::appendVelocityQueue(TwistStampedArray& current_twists,
 }
 
 // Receive a new list of velocities commands and call appendVelocityQueue to insert them into the queue
-// TODO: check that message is sorted properly, default to safe state if anythings wrong?
+// The messages in the array should arrive in chronological order (from message[0] to message[n])
 void AccelerationPlanner::processVelocityCommand(const iarc7_msgs::TwistStampedArray::ConstPtr& message)
 {
 
+    TwistStampedArray::const_iterator checkMessageOrder;
```

Comment:
I originally thought it wouldn't compile if I tried that, but everything went fine